spanner/spanner_samples.rb

```diff
@@ -1873,6 +1873,32 @@ def set_custom_timeout_and_retry project_id:, instance_id:, database_id:
   # [END spanner_set_custom_timeout_and_retry]
 end
 
+def commit_stats project_id:, instance_id:, database_id:
+  # [START spanner_get_commit_stats]
+  # project_id  = "Your Google Cloud project ID"
+  # instance_id = "Your Spanner instance ID"
+  # database_id = "Your Spanner database ID"
+
+  require "google/cloud/spanner"
+
+  spanner = Google::Cloud::Spanner.new project: project_id
+  client  = spanner.client instance_id, database_id
+
+  records = [
+    { SingerId: 1, AlbumId: 1, MarketingBudget: 200_000 },
+    { SingerId: 2, AlbumId: 2, MarketingBudget: 400_000 }
+  ]
+  commit_options = { return_commit_stats: true }
+  resp = client.upsert "Albums", records, commit_options: commit_options
+  puts "Updated data with #{resp.stats.mutation_count} mutations."
+
+  if resp.stats.overload_delay
+    puts "Commit was delayed by #{resp.stats.overload_delay} seconds due to overloaded servers."
+  end
+
```

Comment:
```suggestion
```

Sorry @jiren for the trouble but we were asked to remove this field.

Reply:
@skuruppu removed

Reply:
Thanks a lot @jiren.